sdk/cosmosdb/ci.yml

```diff
@@ -0,0 +1,21 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger: none
+
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - sdk/cosmosdb/cosmos/
+
+jobs:
+  - template: ../../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+    parameters:
+      PackageName: "@azure/cosmos"
+
+  - template: ../../../eng/pipelines/templates/jobs/node6-test-sdk-client.yml
+    parameters:
+      PackagePath: "sdk/cosmosdb/cosmos/"
```

Comment:
Feels like we should figure out if we should make these to templates consistently take the package name instead one being a path and the other being the package name.

Reply:
Yeah - this additional template was a bit of a bolt on. In this PR I won't try to tackle this, this is just setting up so I can get feedback from Azure Pipelines as I start making my other structural changes. Part 2 might address this - its certainly something I'll look at and chat to Brandon about.